client/state/themes/reducer.js

```diff
@@ -162,6 +165,34 @@ export function themeRequests( state = {}, action ) {
 }
 
 /**
+ * Returns the updated Jetpack site wpcom theme install requests state after an action has been
+ * dispatched. The state reflects a mapping of site ID, theme ID pairing to a
+ * boolean reflecting whether a request for the theme install is in progress.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function installThemeOnJetpackRequests( state = {}, action ) {
```

Comment:
This reducer indicates whether install is in progress or not, so probably needs the word `progress` in it.